docs/DeveloperGuide.adoc

```diff
@@ -294,72 +296,225 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a ... |I can ... |So that I ...
+
+|`* * *` |student |add classmate or teammate’s name and contact info |contact them easily
+
+|`* * *` |student |edit classmate or teammate’s name and contact info |Edit them if the information are changed
+
+|`* * *` |student |delete classmate or teammate’s contact |delete if not necessary anymore
+
+|`* * *` |student |categorise contacts into teammate or classmate |
+
+|`* * *` |student |add profile picture to added contact |know at a glance who is the person in my contact
+
+|`* * *` |student |edit profile picture to added contact |edit the picture if changes are necessary
+
+|`* * *` |student |delete profile picture to added contact|
+
+|`* * *` |student |add category specific description |filter out a contact’s remark according to type of contact
+
+|`* *` |student |add contact information on CS2103T tutors |contact them for consultations
+
+|`* *` |student |add contact information on CS2103T professors |can contact them
+
+|`* * *` |student |create meeting events | can keep track of my schedule
+
+|`* * *` |student |create study session events | can keep track of my schedule
+
+|`* * *` |student |create consultation events | can keep track of my schedule
+
+|`* * *` |student |create lesson events | can keep track of my schedule
```

Comment:
It would better if the "so that I..." part is more descriptive and detailed. This can help us to show that the user stories/features provided are very useful and specific for the target user.